services/core-web/common/customAxios.js

```diff
@@ -10,6 +10,11 @@ promiseFinally.shim();
 const UNAUTHORIZED = 401;
 const MAINTENANCE = 503;
 
+const formatErrorMessage = (errorMessage) => {
+  errorMessage = errorMessage.replace("(psycopg2.", "(DatabaseError.");
```

Comment:
is it worth catching "psycopg2" vs "psychopg2." ? It might exist in other places in a string?

Reply:
We can do that, I thought we may just want to change the initial mention of it though. Happy to have to replace all instances if that's what we want?

Reply:
initial mention is probably preferable, so long as it's consistently followed by the period

Reply:
Yup, and by the initial "(".